src/components/message.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+interface UserMessage {
+  message: string;
+}
+
+const Message = (props: UserMessage): any => {
+  return <p>{props.message}</p>;
+};
+
+export default Message;
```

Comment:
If you use function instead of => then you can just declare the function as `export default`. Unfortunately, you can't use `export default` on `const`s in JavaScript.

Reply:
Ohhh, understood. Will do from now on!